lib/test.dart

```diff
@@ -8,6 +8,7 @@ import 'package:path/path.dart' as p;
 
 import 'src/backend/declarer.dart';
 import 'src/backend/invoker.dart';
+import 'src/backend/platform_selector.dart' show PlatformSelector; // for the dartdocs
```

Comment:
If it does cause an "unused import" warning, that should be fixed. Use in DartDoc should count as use.